fedimint-server/src/consensus/mod.rs

```diff
@@ -7,6 +7,7 @@ use std::collections::{BTreeMap, BTreeSet, HashMap, HashSet};
 use std::ffi::OsString;
 use std::iter::FromIterator;
 use std::os::unix::prelude::OsStrExt;
+use std::sync::Mutex;
```

Comment:
Do we want tokio mutex?

Reply:
I don't need to keep it locked across an `await` point, according to the docs it's more lightweight just to use the std one. https://docs.rs/tokio/latest/tokio/sync/struct.Mutex.html

Reply:
Not only that, but also the fact that there should never be contention, so it isn't blocking for too long I assume?

Reply:
It's correct, but the idea of keeping track which Mutex is being used and making sure it's correct is making me sceptical. I'm tempted to ban `std::sync::Mutex` altogether somehow. :D